src/app/main/component/ubs/services/order.service.spec.ts

```diff
@@ -205,4 +206,13 @@ describe('OrderService', () => {
     service.getLiqPayForm();
     expect(service.processLiqPayOrder).toHaveBeenCalled();
   });
+
+  it('method getStepperFinal sould return true', () => {
+    spyOn(service, 'setStepperFinal');
+    service.setStepperFinal(true);
+    setStepperFinalSubjectMock.next(true);
+    setStepperFinalSubjectMock.subscribe((data) => {
+      expect(data).toBeTruthy();
+    });
+  });
```

Comment:
It's a bad unit test
Here you check just if RxJs Subjects work correctly
But you should check `service.yourSubject` to be correct

Reply:
done